recipes/cpc2/baseline/predict.py

```diff
@@ -0,0 +1,107 @@
+"""Make intelligibility predictions from HASPI scores."""
+import json
+import logging
+from pathlib import Path
+from typing import Optional
+
+import hydra
+import numpy as np
+import pandas as pd
+from omegaconf import DictConfig
+from scipy.optimize import curve_fit
+
+from clarity.utils.file_io import read_jsonl
+
+log = logging.getLogger(__name__)
+
+
+class LogisticModel:
```

Comment:
Could this also fit into `clarity/stats.py` and be re-used across subsequent challenges?

Happy to add it to an issue for abstracting code out.

Reply:
Yes, this definitely should as it is equally useful for the CPC1 challenge for future CPC3.